galaxy/templates/deployment-web.yaml

```diff
@@ -42,6 +42,19 @@ spec:
               protocol: TCP
           command: ["/galaxy/server/.venv/bin/uwsgi"]
           args: ["--yaml", "/galaxy/server/config/galaxy.yml"]
+          readinessProbe:
+            httpGet:
+              path: {{ template "galaxy.add_trailing_slash" .Values.ingress.path }}
```

Comment:
Given that this is inside the web pod, should the path be the same as ingress not just `/`?

Reply:
@almahmoud Not sure I follow? This is the same as ingress?

Reply:
What I meant is should the probes be on `/galaxy` where the service is exposed to the outside world (and if so shouldn't it be on port `8090` or `80` then?) or just at `/` on `8080` where it's exposed internally (to my understanding although I could be wrong)?
I talked to Enis briefly about it yesterday and he said that having it on `/galaxy` is likely correct (and the probes do in fact pass). I don't fully understand how it works mapping on `/galaxy` at `8080` though

Reply:
It's running on /galaxy internally too because of this line: https://github.com/CloudVE/galaxy-kubernetes/pull/12/files#diff-bcfeaa7e206ee9dfc036e44b77313133R46

Reply:
the reason for running on the same path internally and externally is to avoid url path rewrites.

Reply:
Ah! That makes sense! Thank you!!!